web_app/api/serializers/user.py

```diff
@@ -27,8 +29,24 @@ class GetUserContractAddressResponse(BaseModel):
     """
     Pydantic model for the get user contract address response.
     """
+
     contract_address: str | None = Field(
         None,
         example="0xabc123...",
         description="The contract address of the user, or None if not deployed.",
     )
+
+
+class GetStats(BaseModel):
```

Comment:
Model name can start with `verb` 
Change it to `StatsData` or `StatsResponse` if it's response model